web_widget_open_tab/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+Edit the tree view and add the widget as the first field, usually, we should use:
+.. code-block:: xml
+
+    <field name="id" widget="open_tab"/>
+
+Now, we can open when clicking with the mouse wheel over the eye.
```

Comment:
A very subjective comment to leave or take ...
I think the eye icon is not the best choice and I would recommend this one:
https://fontawesome.com/icons/external-link-alt?style=solid

Reply:
I'll wait until others give their remark on this! But at least a good suggestion.

Reply:
Good for me

Reply:
@jcdrubay,
I have changed the icon to https://fontawesome.com/v4.7.0/icon/external-link, standard Odoo only supports 4.7.0 icons.
The readme is also adjusted.